tests/integration/bond_test.py

```diff
@@ -648,3 +650,42 @@ def test_set_miimon_100_on_existing_bond(bond99_with_2_slaves):
     bond_config[Bond.OPTIONS_SUBTREE] = {"miimon": 100}
     libnmstate.apply(state)
     assertlib.assert_state_match(state)
+
+
+@pytest.fixture
+def eth1_eth2_with_no_profile():
+    iface_down(ETH1)
+    iface_down(ETH2)
+    yield
+    iface_down(ETH1)
+    iface_down(ETH2)
+
+
+def test_bond_mac_be_consistant_with_diff_slave_order(
+    eth1_eth2_with_no_profile,
+):
+    """
+    When sever reboot, network manager will by default activate slaves in the
+    order of their name. Nmstate should provides the same MAC address for bond
```

Comment:
`Nmstate should provide` instead of `Nmstate should provides`. Other than that, it looks good. Thanks!